API/Utilities/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace API.Utilities
+{
+    public static class ConfigurationExtensions
+    {
+        public static T BindAndValidate<T>(this IConfiguration configuration, string sectionName) where T : new()
```

Comment:
Assuming the logic is the same, I'd definitely prefer a more concrete reference than `where T : new()`. (something like `where T : class` or some clearer type).

Reply:
`where T : class, new()` is also valid, so I went with that

Reply:
Do you have to have the `new()`? I've just never seen it before, feels weird lol.

Reply:
I mean I don't *have* to, that's just what was on the SO post so that's what I went with

I'll remove it if you want LOL

Reply:
I see, I guess it's fine. I looked it up, it's because `: class` allows abstract classes I think.